pkg/provisioner/ironic/ironic.go

```diff
@@ -630,6 +630,12 @@ func (p *ironicProvisioner) InspectHardware() (result provisioner.Result, detail
 	return
 }
 
+func (p *ironicProvisioner) startManualCleaning(ironicNode *nodes.Node, cleanSteps []nodes.CleanStep) (result provisioner.Result, err error){
```

Comment:
This function is only used in one place. Does it need to be a separate function?